Docs: scaffolding for the "Intro" section (#4011)
aiidateam/aiida-core: docs/source/index.rst

Welcome to AiiDA's documentation!
#################################

`AiiDA`_ is an open-source framework implemented in Python that aims to help researchers with managing simple to complex workflows and making them fully reproducible.

.. _AiiDA: https://www.aiida.net

.. panels::
   :centred:

   **Getting Started**

>    New to AiiDA? Check out the getting started guides.

Reviewer: ```suggestion
   First time user or are looking for installation instructions? Start here!
```
To avoid the plagiarism. ;)

Author: closed in c6fdeb39ded24ffaa21d8387d7976ecf1d465749

   .. container:: link-box bottom

      :ref:`To the getting started guides <intro/get_started>`

   ---

   **Tutorial**

>    The tutorials are aimed at new users, that means users who have never worked with AiiDA before or who have only very limited experience using AiiDA in their own problem space.

Reviewer: ```suggestion
   Get your feet wet with a brief introduction to the basics of AiiDA.
```
Let's leave it up to the readers to decide whether that is something they are interested in.

Author: closed in c6fdeb39ded24ffaa21d8387d7976ecf1d465749

   .. container:: link-box bottom

      :ref:`To the tutorials <tutorials>`

   ---

   **How-To Guides**

>    The how-to guides provide concrete goal-oriented recipes for how to achieve certain tasks with AiiDA.

Reviewer: ```suggestion
   Learn how to use AiiDA to power your own work.
```

Author: closed in c6fdeb39ded24ffaa21d8387d7976ecf1d465749

   .. container:: link-box bottom

      :ref:`To the how-to guides <how-to>`

   ---

   **Topics**

>    Background information for those who are already familiar with the AiiDA basics and want to know more about the underlying concepts and design decisions.

Reviewer: This was actually a bit long when I saw it in the browser.
```suggestion
   Comprehensive background information for those who want to learn more about AiiDA's underlying concepts.
```

>    The topics section is understanding-oriented and aimed at users who are generally familiar with AiiDA and provides more in-depth explanations on concepts and design decisions for AiiDA.

Reviewer: ```suggestion
   Background information for those who are already familiar with the AiiDA basics and want to know more about the underlying concepts and design decisions.
```

Author: closed in c6fdeb39ded24ffaa21d8387d7976ecf1d465749

   .. container:: link-box bottom

      :ref:`To the topics <topics>`

   ---

   **API Reference**

>    The reference section is information-oriented and provides a comprehensive description of all components of AiiDAs API, that includes the CLI, the Python API, the RESTful API, and all other APIs which may be implemented in the future.

Reviewer: ```suggestion
   Comprehensive description of all AiiDA components and APIs, including the command-line-interface, the Python interface, and the RESTful API.
```

Author: closed in c6fdeb39ded24ffaa21d8387d7976ecf1d465749
